src/styles/StyleUtils.js

```diff
@@ -297,10 +297,14 @@ function getAnimatedFABStyle(rotate, backgroundColor) {
 
 /**
  * @param {Number} width
- * @param {Number} height
+ * @param {Number | null} height
  * @returns {Object}
  */
-function getWidthAndHeightStyle(width, height) {
+function getWidthAndHeightStyle(width, height = null) {
+    if(isNull(height)) {
+        height = width;
+    }
+
```

Comment:
Not a good idea. It could break things. I would prefer to pass both width and height explicitly as the name say.

Reply:
I disagree. How could this break?

Reply:
This change might break https://github.com/Expensify/App/blob/01e7a70a6ae3573c4b04a394c16c9e088cdd59e6/src/components/AvatarCropModal/AvatarCropModal.js#L326

So because it is not necessary for this PR to work, I will say we don't refactor this function in this PR.

Reply:
@parasharrajat I still don't see how this change would break, and it improves the code, so I think we should leave it in. If you can tell me exactly what's wrong with it, I'd be open to changing it, but just saying it _could_ break isn't enough to prompt a change.

Reply:
Yeah, you are right. So I think the above-posted code only expects the width to be set based on the old implementation because it only passes width. Thus height will be set to `undefined` which will be ignored with the style parser. I haven't tested but after your changes height will also be equal to the width which could break the Avatar crop modal.

Reply:
I agree with your change to this function but my suggestion is that we do not make this change in this PR.

Reply:
I tested the Avatar crop modal and it was fine. 
I insist that we keep this change in. It's an improvement to the code and we are using directly right now. The change is small and easily testable. We can't always make the minimum change possible and expect our code base to stay clean.

Reply:
I don't know if you tested this but it is definitely creating issues. 
![image](https://user-images.githubusercontent.com/24370807/205732510-2cc56a69-9ddc-46fa-9454-b6ef9caab508.png)


Notice the gap above between the image canvas and the zoom slider. 


On a side note, IMO we shouldn't make this function set the height automatically. I do not have a logical explanation for it but I think it is better this way. 


If you insist, I will approve this as well after the above issue is fixed.

Reply:
This was a great opportunity to fix that line too. It's strange that we were setting `height: undefined` intentionally. Better to just not set it, right?